integration/container_tty_leak_linux_test.go

```diff
@@ -0,0 +1,144 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package integration
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/url"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/containerd/containerd/v2/integration/images"
+	"github.com/containerd/containerd/v2/pkg/namespaces"
+	"github.com/stretchr/testify/require"
+	"k8s.io/client-go/rest"
+	remoteclient "k8s.io/client-go/tools/remotecommand"
+	criruntime "k8s.io/cri-api/pkg/apis/runtime/v1"
```

Comment:
We are using an older version of cri-api in 1.6 branch. Ref: https://github.com/containerd/containerd/blob/release/1.6/go.mod#L77.

cc: @fuweid @henry118

Reply:
> We are using an older version of cri-api in 1.6 branch. Ref: https://github.com/containerd/containerd/blob/release/1.6/go.mod#L77.
> 
> cc: @fuweid @henry118

I'll look into this next week.